test/evergreen/configure_combinations.sh

```diff
@@ -1,59 +1,84 @@
-CC=clang CFLAGS=\"-Wall -Werror -Qunused-arguments -Wno-self-assign -Wno-parentheses-equality -Wno-array-bounds\""
+flags="-DCMAKE_TOOLCHAIN_FILE=$curdir/cmake/toolchains/gcc.cmake -DCMAKE_C_FLAGS=\"-Werror -Wall -Wextra -Waddress -Waggregate-return -Wbad-function-cast -Wno-missing-field-initializers -Wno-cast-function-type -Wcast-align -Wformat-security -Wformat-nonliteral -Wformat=2 -Wnested-externs -Wno-unused-parameter -Wpointer-arith -Wredundant-decls  -Wundef -Wunused -Wwrite-strings -O -fno-strict-aliasing -Wuninitialized -Wno-discarded-qualifiers -Wno-incompatible-pointer-types -Wno-int-conversion -Wno-sign-conversion -Wno-pointer-sign -Wno-unused-variable\"
+-DCMAKE_TOOLCHAIN_FILE=$curdir/cmake/toolchains/clang.cmake -DCMAKE_C_FLAGS=\"-Wall -Werror -Qunused-arguments -Wno-self-assign -Wno-parentheses-equality -Wno-array-bounds\""
 
-options="--enable-diagnostic
---disable-shared
---disable-static --enable-python
---enable-snappy --enable-zlib --enable-lz4
---with-builtins=lz4,snappy,zlib
---enable-diagnostic --enable-python
---enable-strict --disable-shared"
+options="-DHAVE_DIAGNOSTIC=1
+-DENABLE_SHARED=0 -DENABLE_STATIC=1
+-DENABLE_STATIC=0 -DENABLE_PYTHON=1
+-DENABLE_SNAPPY=1 -DENABLE_ZLIB=1 -DENABLE_LZ4=1
+-DHAVE_BUILTIN_EXTENSION_LZ4=1 -DHAVE_BUILTIN_EXTENSION_SNAPPY=1 -DHAVE_BUILTIN_EXTENSION_ZLIB=1
+-DHAVE_DIAGNOSTIC=1-DENABLE_PYTHON=1
+-DENABLE_STRICT=1 -DENABLE_STATIC=1 -DENABLE_SHARED=0 -DWITH_PIC=1"
 
 saved_IFS=$IFS
 cr_IFS="
 "
 
 # This function may alter the current directory on failure
 BuildTest() {
-        extra_config=--enable-silent-rules
         echo "Building: $1, $2"
         rm -rf ./build || return 1
         mkdir build || return 1
         cd ./build
-        eval ../configure $extra_config "$1" "$2" \
-                 --prefix="$insdir" || return 1
-        eval make "$3" || return 1
-        make -C examples/c check VERBOSE=1 > /dev/null || return 1
-        case "$2" in
-                # Skip the install step with Python.  Even with --prefix, the
-                # install tries to write to /usr/lib64/python2.7/site-packages .
-                *enable-python* )  doinstall=false;;
-                # Non-shared doesn't yet work: library is not found at link step (??)
-                *disable-shared* ) doinstall=false;;
-                * )                doinstall=true;;
-        esac
-        if $doinstall; then
-                eval make install || return 1
-                cflags=`pkg-config wiredtiger --cflags --libs`
-                [ "$1"  == "CC=clang" ] && compiler="clang" || compiler="cc"
-                echo $compiler -o ./smoke ../examples/c/ex_smoke.c $cflags
-                $compiler -o ./smoke ../examples/c/ex_smoke.c  $cflags|| return 1
-                LD_LIBRARY_PATH=$insdir/lib ./smoke || return 1
+        eval $CMAKE_BIN $extra_config "$1" "$2" \
```

Comment:
Is it intentional the setting line for `$extra_config` gets removed?

Reply:
Yep, I removed `extra_config` since `--enable-silent-rules` is not necessary with CMake. Given that was its only use, I figured we didn't need it. Though I missed cleaning up the variable on this line. Fixed in 7309abee8efd0e26251f0151070d253f8a6f507c